types/context.go

```diff
@@ -335,30 +336,30 @@ func (c Context) Value(key interface{}) interface{} {
 // Store / Caching
 // ----------------------------------------------------------------------------
 
-// KVStore fetches a KVStore from the MultiStore.
-func (c Context) KVStore(key storetypes.StoreKey) storetypes.KVStore {
-	return gaskv.NewStore(c.ms.GetKVStore(key), c.gasMeter, c.kvGasConfig)
+// KVStore fetches a KVStore from the RootStore.
+func (c Context) KVStore(key string) store.KVStore {
+	return gaskv.New(c.rs.GetKVStore(key), c.gasMeter, c.kvGasConfig)
 }
 
-// TransientStore fetches a TransientStore from the MultiStore.
-func (c Context) TransientStore(key storetypes.StoreKey) storetypes.KVStore {
-	return gaskv.NewStore(c.ms.GetKVStore(key), c.gasMeter, c.transientKVGasConfig)
-}
+// // TransientStore fetches a TransientStore from the MultiStore.
+// func (c Context) TransientStore(key storetypes.StoreKey) storetypes.KVStore {
+// 	return gaskv.NewStore(c.ms.GetKVStore(key), c.gasMeter, c.transientKVGasConfig)
+// }
 
-// CacheContext returns a new Context with the multi-store cached and a new
-// EventManager. The cached context is written to the context when writeCache
+// BranchContext returns a new Context with the RootStore branched and a new
+// EventManager. The branched context is written to the context when commitBranch
 // is called. Note, events are automatically emitted on the parent context's
 // EventManager when the caller executes the write.
-func (c Context) CacheContext() (cc Context, writeCache func()) {
```

Comment:
id leave this as is if we want to test on mainnets, we wont be able to change modules that depend on this